WireDomain/Sources/WireDomain/Event Processing/TeamEventProcessor/TeamMemberUpdateEventProcessor.swift

```diff
@@ -32,9 +33,24 @@ protocol TeamMemberUpdateEventProcessorProtocol {
 
 struct TeamMemberUpdateEventProcessor: TeamMemberUpdateEventProcessorProtocol {
 
-    func processEvent(_: TeamMemberUpdateEvent) async throws {
-        // TODO: [WPB-10186]
-        assertionFailure("not implemented yet")
+    enum Error: Swift.Error, Equatable {
+        case failedToFindMember(_ membershipId: UUID)
```

Comment:
There is a SwiftFormat rule for that, but it's dangerous to apply it on an existing codebase.
I would actually prefer to have it enabled, but we would have to be careful.
(it can break Codable for example)
https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#acronyms